proposals/vitess.adoc

```diff
@@ -0,0 +1,109 @@
+
+*Sponsor / Advisor from TOC*: Brian Grant <briangrant@google.com>
+
+*Unique Identifier*: vitess
+
+*License*: ALv2
+
+*Maturity Level:* Incubating
+
+*Source control repositories*: https://github.com/youtube/vitess
+
+*Initial Committers*:
+
+* Sugu Sougoumarane
+* Mike Solomon
+* Alain Jobart
+* Anthony Yeh
+* Michael Berlin
+
+*Infrastructure requirements*: CI, CNCF Cluster
+
+*Issue tracker*: https://github.com/youtube/vitess/issues
+
+*Mailing lists*
+
+* https://vitess.slack.com (see https://vitess.slackarchive.io)
+* https://groups.google.com/forum/#!forum/vitess
+
+*Website*: http://vitess.io
+
+*Release methodology and mechanics*
+
+The current release is v2.1.1. We use the link:http://semver.org/[semantic versioning] numbering system. The releases are announced using link:https://github.com/youtube/vitess/releases[github’s releases] feature and link:http://blog.vitess.io/2017/04/vitess-releases-version-21.html[through a blog post]. Additionally, a release involves tagging of a docker image and a Maven release of the Java client. The next major release changes the client protocols, and will be tagged 3.0.
+
+The master branch is kept strictly stable, and we encourage users to regularly pull and deploy from it, as YouTube does. Many users have adopted this approach.
+
+*Social media accounts*: N/A
+
+*Existing sponsorship*:
+
+* Google (lead)
+* Hubspot (many core contributions)
+* Slack (many core contributions)
+* Flipkart (Java client)
+* BetterCloud (TLS support)
+* Pixel Federation (PHP client)
+
+*Adopters*: Source: https://github.com/youtube/vitess/blob/master/ADOPTERS.md
+
+* Axon
+* BetterCloud
+* FlipKart
+* HubSpot
+* Nozzle
+* Pixel Federation
+* Quiz of Kings
+* Slack
+* Square
+* Stitch Labs
+* YouTube
+
+*Statement on alignment with CNCF mission*:
+
+NoSQL storage systems were designed to scale out, but focus on unstructured and non-transactional data. However, it is complex to migrate or build applications that truly need transactions, indexes, and joins over structured data using NoSQL. NewSQL storage systems such as Vitess fill that gap, and enable more applications to migrate to cloud-native architectures and to scale out. Vitess was built to be cloud-native for use within Google, and can link:http://vitess.io/getting-started/[run on Kubernetes].
```

Comment:
@sougou 

How do you feel about replacing "NewSQL storage systems such as Vitess" with the following:

"NewSQL storage systems and database orchestration systems such as Vitess"

?

We changed the "storage system" terminology in the description at the top, but we missed it here.

Reply:
I can change it to "Database orchestration systems such as Vitess".

No need to mention NewSQL at all.

Reply:
Done.